tests/no_seq/README.rst

```diff
@@ -0,0 +1,28 @@
+Forced non-sequential relations to an input
```

Comment:
I agree that this feature *probably* needed. I think you need to expand on *why* it is needed in human english. It also needs a significantly simpler example (rather than the *whole* quick logic logic cell) -- hopefully only 1-2 cells.

Reply:
I've reworked the example (simplified it).